js/flightlog.js

```diff
@@ -303,6 +303,35 @@ function FlightLog(logData) {
         return numMotors;
     };
 
+    /*
+     * Propagates array data fiels over empty fields
+     * Debug fields can be arrays with data, so it fills data gaps when no arrays are stored
+     */
+    function propagateArrayFields(chunks)
+    {
```

Comment:
I'm not too sure about this. I think in the rest of the code we simply store the latest value, and add them when we process other kind of data, to add the values while processing. Maybe is clear with your method, with a little more process time (it needs to look at all the data chunk). 

Can you look at the slow frames and gps frames chunks to see if they can be "filled" with this and remove the old implementation? Taking times of this method will be great to see if we loose too much time. If it takes not too much time, I prefer your implementation.

When there are errors in the blackbox (frames lost), this method will fill the data? I think remember that not, that we don't have data in the chunk when the frames are lost.

Reply:
> Can you look at the slow frames and gps frames chunks to see if they can be "filled" with this and remove the old implementation? Taking times of this method will be great to see if we loose too much time. If it takes not too much time, I prefer your implementation.

I have been looking for the current implementation. Is it this?
![imagen](https://user-images.githubusercontent.com/2889851/219936584-16c87806-46d4-48e8-b6ab-ca4d21a672a8.png)


> When there are errors in the blackbox (frames lost), this method will fill the data? I think remember that not, that we don't have data in the chunk when the frames are lost.
Still looking into this

Reply:
Has this been resolved?

Reply:
@McGiverGim are you okay with this - for a follow up PR?